frontend/website/docs/index.md

```diff
@@ -11,3 +11,17 @@ This breakthrough is made possible due to zk-SNARKs - a type of succinct cryptog
 Trying out Coda is simple - head over to the [Getting Started page](/docs/getting-started/) to install the Coda client and join the network.
 
 You can also sign up for the public testnet updates [here](https://bit.ly/TestnetForm).
+
+<span id="testnet-points" />
+## Testnet Points[\*](#disclaimer)
```

Comment:
you don't need to do this, all headers automatically get tagged - this one will be `#testnet-points`

Reply:
I think there may be weirdness with the `\*` which is why I wanted to make it explicit just in case